src/relay/transforms/fake_quantization_to_integer.cc

```diff
@@ -28,42 +28,52 @@
 #include <tvm/relay/expr_functor.h>
 #include <tvm/relay/transform.h>
 
+#include "../qnn/op/make_qnn_op.h"
+
 namespace tvm {
 namespace relay {
 
 /* Description of FakeQuantizationToInteger
  *
- * The purpose of this pass is to find regions of the graph that follow
- * the general pattern:
+ * The purpose of this pass is to find regions of the graph that can be quantized and convert all
+ * ops in this subgraph to quantized ops. E.g. for below example nodes dq1, dq2, op1 and op2 will
+ * participate in transformation to quantized ops while op3 has non quantized input and stays out of
+ * transformation scope:
  *
  *   x    w
  *   |    |
- *   dq   dq
+ *   dq1   dq2
  *    \   /
- *     op1
+ *     op1 (can be quantized)
  *      |
- *     op2
+ *     op2 (can be quantized)     <- anchor node to be identified and used in mutation pass
  *      |
- *      q
+ *      |       non_quantized_input
+ *      \     /
+ *        op3    <- cannot be quantized
```

Comment:
I don't like this approach, it solves some issues where we don't support integer ops embedded in larger devices, but has the property of mis-representing the user's original intention for segments of the graph that might have actually been dequantized.